packages/recurring-payments-job/src/__tests__/recurring-payments-processor.spec.js

```diff
@@ -70,9 +70,26 @@ const getMockDueRecurringPayment = ({ agreementId = 'test-agreement-id', id = 'a
   expanded: { activePermission: { entity: { referenceNumber } } }
 })
 
+const makeSendPaymentResponse = (overrides = {}) => ({
+  payment_id: 'pay-1',
+  created_date: '2025-01-01T00:00:00.000Z',
+  ...overrides
+})
+
 describe('recurring-payments-processor', () => {
   const [{ value: debugLogger }] = db.mock.results
 
+  const date1 = '2025-01-01T00:00:00.000Z'
+  const date2 = '2025-01-01T00:01:00.000Z'
+
+  const mockTwoDuePayments = () => {
+    salesApi.getDueRecurringPayments.mockResolvedValueOnce([getMockDueRecurringPayment(), getMockDueRecurringPayment()])
+
+    salesApi.preparePermissionDataForRenewal.mockResolvedValue({ licensee: { countryCode: 'GB-ENG' } })
```

Comment:
Best to do `mockResolvedValueOnce` over `mockResolvedValue` so you don't have to worry about mocks spilling over to other tests.